.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
+  "printWidth": 120,
+  "singleQuote": true,
```

Comment:
When I was modifying tests I couldn't figure out what settings were used when formatting, decided to add some to make it more consistent. Any suggestions on what you guys want to use?